astropy/stats/tests/test_funcs.py

```diff
@@ -346,6 +346,19 @@ def test_mad_std():
         data = normal(5, 2, size=(100, 100))
         assert_allclose(funcs.mad_std(data), 2.0, rtol=0.05)
 
+def test_mad_std_with_axis():
+    with NumpyRNGContext(12345):
```

Comment:
You actually don't need to use the `NumpyRNGContext` here because you define the array with actual values.